### PR TITLE
simple additions

### PR DESCRIPTION
**Pull request by FiendFyre5**
Added a reaction and head or Tails

### CODE REVIEW

**Comment by JavaCafe01** [comment]
@FiendFyre5 the command is gonna be `-destiny`, is that alright?

**Comment by FiendFyre5** [comment]
I couldn't think of something else lol. Do you have any suggestions?

**Comment by JavaCafe01** [comment]
> I couldn't think of something else lol. Do you have any suggestions?

`-8ball`? And you can do multiple yes phrases and mult no phrases. Or do you want to keep it as a coin?

**Comment by FiendFyre5** [comment]
Maybe for later. Right now I'm good with coin

File: bot.py [hunk 1]
```diff
@@ -130,7 +140,8 @@ async def on_message(message):
     await react_message("historia", message, bot.get_emoji(anger_emoji))
     await react_message("levi", message, knife_emoji)
     await react_message("eren", message, bot.get_emoji(creepy_annie_emoji))
-
+    await react_message("gokul", message, bot.get_emoji(shoe))
```
Comment (JavaCafe01) [inline bot.py:143]: @FiendFyre5 because the shoe is unicode, you dont need the `get_emoji` method

**Review by JavaCafe01 — CHANGES_REQUESTED** [review]

File: bot.py (hunk 1 above)
Comment (FiendFyre5) [reply bot.py:143]: My bad

**Review by JavaCafe01 — APPROVED** [review]

File: bot.py (hunk 1 above)
Comment (FiendFyre5) [reply bot.py]: Pushed the fixed commit

**Comment by FiendFyre5** [comment]
Yay! Thanks :)

**Comment by JavaCafe01** [comment]
you forgot to import random

**Pull request merged**